sql/core/src/test/scala/org/apache/spark/sql/ExplainSuite.scala

```diff
@@ -332,4 +335,57 @@ class ExplainSuite extends QueryTest with SharedSparkSession with DisableAdaptiv
   }
 }
 
+class ExplainSuiteAE extends ExplainSuiteHelper with EnableAdaptiveExecutionSuite {
+  import testImplicits._
+
+  test("Explain formatted") {
+    val df1 = Seq((1, 2), (2, 3)).toDF("k", "v1")
+    val df2 = Seq((2, 3), (1, 1)).toDF("k", "v2")
+    val testDf = df1.join(df2, "k").groupBy("k").agg(count("v1"), sum("v1"), avg("v2"))
+    // trigger the final plan for AQE
+    testDf.collect()
+    // whitespace
+    val ws = " "
+    //   == Physical Plan ==
+    //   AdaptiveSparkPlan (14)
```

Comment:
@Ngone51, it would be better if we show the before/after results in the PR description. What was it before out of curiosity? Was it just `AdaptiveSparkPlan (14)`?

Reply:
> Was it just AdaptiveSparkPlan (14)?
Yeah..have updated description, thanks!